oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/base/MetricsDAO.java

```diff
@@ -57,11 +62,47 @@ public MetricsDAO(InfluxClient client, StorageBuilder<Metrics> storageBuilder) {
     }
 
     @Override
-    public List<Metrics> multiGet(Model model, List<String> ids) throws IOException {
-        final WhereQueryImpl<SelectQueryImpl> query = select()
-            .raw(ALL_FIELDS)
-            .from(client.getDatabase(), model.getName())
-            .where(contains("id", Joiner.on("|").join(ids)));
+    public List<Metrics> multiGet(Model model, List<Metrics> metrics) throws IOException {
+        final TableMetaInfo metaInfo = TableMetaInfo.get(model.getName());
+        final String queryStr;
+        if (model.getName().endsWith("_traffic")) {
```

Comment:
Don't like this, but seems don't have much better idea. Adding an interface seems not vey helpful. cc @kezhenxu94